### PR TITLE
[ISSUE #377]Translation for "数据集成-RocketMQ Connect 概览" Section in the v4.x & 5.0 Document (CN -> EN) 

### LINKED ISSUE #377: Translation for "数据集成-RocketMQ Connect 概览" Section in the v4.x & 5.0 Document (CN -> EN)
The Chinese version of the "数据集成-RocketMQ Connect 概览" section in the v4.x & v5.0 document has now been finished. However, its English version is still incomplete.

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #377 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/25RocketMQ Connect Overview.md [hunk 1]
```diff
@@ -1,42 +1,40 @@
-# RocketMQ Connect 概览
+# RocketMQ Connect overview
 
-RocketMQ Connect是RocketMQ数据集成重要组件，可将各种系统中的数据通过高效，可靠，流的方式，流入流出到RocketMQ，它是独立于RocketMQ的一个单独的分布式，可扩展，可容错系统，
-它具备低延时，高可靠性，高性能，低代码，扩展性强等特点，可以实现各种异构数据系统的连接，构建数据管道，ETL，CDC，数据湖等能力。
+RocketMQ Connect is an important component of RocketMQ data integration, which can efficiently, reliably, and flowingly flow data in and out of RocketMQ from various systems. It is a separate, distributed, scalable, and fault-tolerant system that has low latency, high reliability, high performance, low code, and strong scalability. It can achieve various heterogeneous data system connections, data pipeline building, ETL, CDC, and data lake capabilities.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/25RocketMQ Connect Overview.md]: > RocketMQ Connect is an important component of RocketMQ data integration, which can efficiently, reliably, and flowingly flow data in and out of RocketMQ from various systems.

RocketMQ Connect is an important component of RocketMQ data integration, which can transfer data in and out of RocketMQ from various systems efficiently and reliably.

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/25RocketMQ Connect Overview.md [hunk 2]
```diff
@@ -1,42 +1,40 @@
...
 
 ![RocketMQ Connect部署模型集群](../picture/32rocketmq-connect/deploy3.png)
 
-单机模式，Connector任务运行在单机上，Worker本身没有高可用，任务offset信息持久化在本地。适合一些对高可没有什么要求或者不需要Worker保障高可用的场景，例如部署在k8s集群中，由k8s集群保障高可用。
+In standalone mode, Connector tasks run on a single machine and Worker itself does not have high availability, task offset information is persisted locally. It is suitable for scenarios where there is no high availability requirement or does not require Worker to ensure high availability, such as deployment in k8s clusters, which are guaranteed by k8s clusters.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/25RocketMQ Connect Overview.md]: k8s -> K8s

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
Hey @mxsm , well done. Only some minor mistakes need to be fixed. Remember to correct them in the 5.0 version as well. Thx.

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/25RocketMQ Connect Overview.md (hunk 2 above)
Comment (mxsm) [reply i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/25RocketMQ Connect Overview.md]: I will fix it later @tsunghanjacktsai

**Comment by mxsm** [comment]
@tsunghanjacktsai  PTAL~

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**